core/workspace_svg.js

```diff
@@ -190,6 +190,16 @@ Blockly.WorkspaceSvg.prototype.startScrollY = 0;
  */
 Blockly.WorkspaceSvg.prototype.dragDeltaXY_ = null;
 
+/***
+ * Distance to scroll when a mouse wheel event is received and
+ * its delta mode is line (0x1) instead of pixel (0x0). In these
+ * cases, a single "scroll" has a delta of 1, which makes the
+ * workspace scroll very slowly (just one pixel). To compensate,
+ * that delta is multiplied by this value.
+ * @type {number}
+ */
+Blockly.WorkspaceSvg.prototype.lineScrollMultiplier = 15;
+
```

Comment:
Is there a better place to put this constant, or is putting it here right?

Reply:
Constants.js is a good place for this.

Add @package and @const annotations when you put it there, and give it an all caps name to match conventions for constants.